### DISCUSSION TITLE
Tailwind CSS v3.0

### DISCUSSION THREAD

**Discussion by bergser** [tw-in-js/twind#230] [Q&A] [answered]
The [new Tailwind version 3.0](https://github.com/tailwindlabs/tailwindcss/releases) was released on Dec 9. Are there any plans to upgrade `twind` as well? So that the latest tailwind features can be used

**Comment by sastan** [comment, marked as answer]
I will create a v0.17 release in the next weeks. I aim for christmas. It will include the new and changed utilities, the color opacity shorthand, the new variants. Maybe the arbitrary css ([color:red]).

**Reply by bergser** [reply]
That's really good news! Thx for fast reply

**Comment by Ethaan** [comment]
If help needed, open to help on the PR, wish i have an idea on where to start, but happy to help if a WIP PR is ever pushed

**Reply by sastan** [reply]
Thanks! I try to create a kick-off PR where you can jump in. Christmas didn't work for v0.17.  Sorry — I just do not find the time at the moment.

**Comment by sastan** [comment]
Twind v1 is around the corner with full Tailwind v3 compatibilty.

Please give it a try. Here are some links to get you started:

- [The v1 branch](https://github.com/tw-in-js/twind/tree/next)
- [Reference](https://github.com/tw-in-js/twind/blob/next/documentation/40-reference/README.md) until the website is ready
- [Examples](https://github.com/tw-in-js/twind/tree/next/examples) — the [CDN example](https://github.com/tw-in-js/twind/tree/next/examples/twind-cdn) could be a good start

**Reply by bstro** [reply]
The link to the reference appears to be broken now—where might this information be accessible?

**Reply by sastan** [reply]
Sorry for that. Working on the docs.... Here its is https://github.com/tw-in-js/twind/blob/next/documentation/reference.md
